internal/docs/docs-components/code-block.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react'
 import styled from 'styled-components'
+import containerStyles from '../../../core/components/_helpers/container-styles'
```

Comment:
nuuuuu, this breaks whenever make the tiniest change to folder structure

```suggestion
import containerStyles from '@auth0/cosmos/_helpers/container-styles'
```

This way, it's more reliable